web/app/css/service-mesh.css

```diff
@@ -25,6 +25,10 @@
   border-radius: var
 }
 
+.service-mesh-table {
+  margin-top: 18px;
+}
+
```

Comment:
This is a big improvement. I wonder if we want to do the same for `.incomplete-mesh-message, .complete-mesh-message`. We could add a 32px top margin to those classes to line them up with the header of the Data Plane table. This is how it looks now:

![image](https://user-images.githubusercontent.com/9226/35585670-762e2382-05ad-11e8-917a-c22e19c2e7b0.png)

And with 32px padding:

![image](https://user-images.githubusercontent.com/9226/35585779-bf5d452e-05ad-11e8-9303-d07b46d8a51d.png)

What do you think?

Reply:
yeah, that looks better to me! will do!